website/src/content/blog/2025-03-31-typespec-1-0-release/type_spec_1_0_release.md

```diff
@@ -0,0 +1,188 @@
+- **<a href="https://www.npmjs.com/package/@typespec/openapi3" target="_blank" rel="noopener noreferrer">@typespec/openapi3</a>**: 1.0.0-rc.0
+- **<a href="https://www.npmjs.com/package/@typespec/json-schema" target="_blank" rel="noopener noreferrer">@typespec/json-schema</a>**: 1.0.0-rc.0
+
+These RC packages have undergone extensive testing and are approaching their stable release. While we recommend evaluating them in production scenarios, be aware that minor changes could still occur before the final 1.0 release.
+
+TypeSpec 1.0-RC includes several additional components in preview state that are still undergoing development and testing. We would greaty appreciate your feedback on these features as we work towards their final release.
+
+### Additional protocol emitters (Preview)
+
+- **<a href="https://www.npmjs.com/package/@typespec/protobuf" target="_blank" rel="noopener noreferrer">@typespec/protobuf</a>**: Generates Protocol Buffer (protobuf) definitions
+- **<a href="https://www.npmjs.com/package/@typespec/rest" target="_blank" rel="noopener noreferrer">@typespec/rest</a>**: Produces JSON descriptions of REST APIs
+
+### Libraries (Preview)
+
+- **<a href="https://www.npmjs.com/package/@typespec/events" target="_blank" rel="noopener noreferrer">@typespec/events</a>**
+- **<a href="https://www.npmjs.com/package/@typespec/rest" target="_blank" rel="noopener noreferrer">@typespec/rest</a>**
+- **<a href="https://www.npmjs.com/package/@typespec/sse" target="_blank" rel="noopener noreferrer">@typespec/sse</a>**
+- **<a href="https://www.npmjs.com/package/@typespec/streams" target="_blank" rel="noopener noreferrer">@typespec/streams</a>**
+- **<a href="https://www.npmjs.com/package/@typespec/versioning" target="_blank" rel="noopener noreferrer">@typespec/versioning</a>**
+- **<a href="https://www.npmjs.com/package/@typespec/xml" target="_blank" rel="noopener noreferrer">@typespec/xml</a>**
+
+### Client/server code emitters (Preview)
+
+- **<a href="https://www.npmjs.com/package/@typespec/http-client-csharp" target="_blank" rel="noopener noreferrer">@typespec/http-client-csharp</a>**
+- **<a href="https://www.npmjs.com/package/@typespec/http-client-js" target="_blank" rel="noopener noreferrer">@typespec/http-client-js</a>**
+- **<a href="https://www.npmjs.com/package/@typespec/http-client-java" target="_blank" rel="noopener noreferrer">@typespec/http-client-java</a>**
+- **<a href="https://www.npmjs.com/package/@typespec/http-client-python" target="_blank" rel="noopener noreferrer">@typespec/http-client-python</a>**
+- **<a href="https://www.npmjs.com/package/@typespec/http-server-csharp" target="_blank" rel="noopener noreferrer">@typespec/http-server-csharp</a>**
+- **<a href="https://www.npmjs.com/package/@typespec/http-server-js" target="_blank" rel="noopener noreferrer">@typespec/http-server-js</a>**
+
+## Getting started with TypeSpec 1.0-RC
```

Comment:
I would put this as the first section after the intro paragraph, personally. Call to action right away is always a good idea 😁

Reply:
Based on our earlier round of feedback I'm going to leave it as is, so it's clear what we're shipping before asking people to try it.